From aacc3e8e00f528224df0b7ebcaabe06d60ce81a1 Mon Sep 17 00:00:00 2001
From: Eugen Hristev <eugen.hristev@microchip.com>
Date: Mon, 20 Dec 2021 13:54:22 +0200
Subject: [PATCH 328/471] ARM: dts: at91: sama7g5ek: enable SDMMC high speed
 modes

SDMMC0 works up to HS200 on this board with non-removable eMMC 5.1 flash.
SDMMC1 works up to DDR50 on this board with SD-Cards.

Signed-off-by: Eugen Hristev <eugen.hristev@microchip.com>
---
 arch/arm/boot/dts/at91-sama7g5ek.dts | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/at91-sama7g5ek.dts b/arch/arm/boot/dts/at91-sama7g5ek.dts
index a64afaee0baa..2601d1aba8da 100644
--- a/arch/arm/boot/dts/at91-sama7g5ek.dts
+++ b/arch/arm/boot/dts/at91-sama7g5ek.dts
@@ -642,8 +642,10 @@ &rtt {
 &sdmmc0 {
 	bus-width = <8>;
 	non-removable;
-	no-1-8-v;
-	sdhci-caps-mask = <0x0 0x00200000>;
+	mmc-ddr-1_8v;
+	cap-mmc-highspeed;
+	cap-mmc-hw-reset;
+	mmc-hs200-1_8v;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdmmc0_default>;
 	status = "okay";
@@ -651,8 +653,6 @@ &sdmmc0 {
 
 &sdmmc1 {
 	bus-width = <4>;
-	no-1-8-v;
-	sdhci-caps-mask = <0x0 0x00200000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdmmc1_default>;
 	status = "okay";
@@ -661,7 +661,6 @@ &sdmmc1 {
 &sdmmc2 {
 	bus-width = <4>;
 	no-1-8-v;
-	sdhci-caps-mask = <0x0 0x00200000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_sdmmc2_default>;
 };
-- 
2.34.1

